charts/ccd-api-gateway-web/values.aat.template.yaml

```diff
@@ -1,3 +1,5 @@
 nodejs:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
+  environment:
+    CORS_ORIGIN_WHITELIST: "*"
```

Comment:
Why are we allowing any domain in the CORS whitelist? Isn't it contrary to the purpose of the fix attempt?